great_expectations/rule_based_profiler/data_assistant/column_value_missing_data_assistant.py

```diff
@@ -29,9 +29,9 @@
 
 if TYPE_CHECKING:
     from great_expectations.rule_based_profiler.domain_builder import DomainBuilder
-    from great_expectations.rule_based_profiler.parameter_builder import (
-        ParameterBuilder,
-    )
+from great_expectations.rule_based_profiler.parameter_builder import (
+    ParameterBuilder,
+)
```

Comment:
@TrangPham How come this no longer works under `TYPE_CHECKING' only?  If we can move it back in, it would be advantageous for code clarity (i.e., "this is just for type hinting").  Thanks.